### PR TITLE
Add docker/ to push daemon image for deployment

### PR DESCRIPTION
**Pull request by yunyeng**
This change is [Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/266)

### CODE REVIEW

**Review by efoxepstein — CHANGES_REQUESTED** [review]
> Reviewed 4 of 4 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/266)* status: all files reviewed, 1 unresolved discussion (waiting on @SanjayVas, @stevenwarejones, @wangyaopw, and @yunyeng)

___
*[build/variables.bzl, line 17 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/266#-MuRhLQm-ZTQgCRkZzD8:-MuRhLQm-ZTQgCRkZzD9:bulygo3) ([raw file](https://github.com/world-federation-of-advertisers/panel-exchange-client/blob/b7ec38c79312fddad853afad77579f7cbe3edba0/build/variables.bzl#L17)):*
> ```Starlark
> """
> This file contains local settings for building and deploying the codebase.
> It is in .gitignore; please don't commit your local changes.
> ```

Is it?

**Review by wangyaopw — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/266)* status: all files reviewed, 1 unresolved discussion (waiting on @efoxepstein, @SanjayVas, and @stevenwarejones)

___
*[build/variables.bzl, line 17 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/266#-MuRhLQm-ZTQgCRkZzD8:-MuRiHRM8B0meRQDz1Dc:b-de49z6) ([raw file](https://github.com/world-federation-of-advertisers/panel-exchange-client/blob/b7ec38c79312fddad853afad77579f7cbe3edba0/build/variables.bzl#L17)):*
<details><summary><i>Previously, efoxepstein (Eli Fox-Epstein) wrote…</i></summary><blockquote>

Is it?
</blockquote></details>

No longer true now.

All these are mark vars, and we don't need to update this file in local branch anymore.

**Review by wangyaopw — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/266)* status: all files reviewed, 1 unresolved discussion (waiting on @efoxepstein, @SanjayVas, and @stevenwarejones)

___
*[build/variables.bzl, line 17 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/266#-MuRhLQm-ZTQgCRkZzD8:-MuRiQxzCMtkhU8QANCC:bibglcw) ([raw file](https://github.com/world-federation-of-advertisers/panel-exchange-client/blob/b7ec38c79312fddad853afad77579f7cbe3edba0/build/variables.bzl#L17)):*
<details><summary><i>Previously, wangyaopw (Yao Wang) wrote…</i></summary><blockquote>

No longer true now.

All these are mark vars, and we don't need to update this file in local branch anymore.
</blockquote></details>

In other words, we can remove this comment.

**Review by efoxepstein — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/266)* status: all files reviewed, 1 unresolved discussion (waiting on @SanjayVas, @stevenwarejones, and @yunyeng)

___
*[build/variables.bzl, line 17 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/266#-MuRhLQm-ZTQgCRkZzD8:-MuRjIqGE7TR4I_ZMMeR:b-ac5ax5) ([raw file](https://github.com/world-federation-of-advertisers/panel-exchange-client/blob/b7ec38c79312fddad853afad77579f7cbe3edba0/build/variables.bzl#L17)):*
<details><summary><i>Previously, wangyaopw (Yao Wang) wrote…</i></summary><blockquote>

In other words, we can remove this comment.
</blockquote></details>

Great -- @yunyeng, please remove the comment

**Review by yunyeng — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/266)* status: 3 of 4 files reviewed, 1 unresolved discussion (waiting on @efoxepstein, @SanjayVas, and @stevenwarejones)

___
*[build/variables.bzl, line 17 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/266#-MuRhLQm-ZTQgCRkZzD8:-MuRjp7J05C9T9PDQD7B:b-896fix) ([raw file](https://github.com/world-federation-of-advertisers/panel-exchange-client/blob/b7ec38c79312fddad853afad77579f7cbe3edba0/build/variables.bzl#L17)):*
<details><summary><i>Previously, efoxepstein (Eli Fox-Epstein) wrote…</i></summary><blockquote>

Great -- @yunyeng, please remove the comment
</blockquote></details>

Done.

**Review by efoxepstein — APPROVED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/266)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @SanjayVas and @stevenwarejones)

**Review by efoxepstein — APPROVED** [review]
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/266)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @SanjayVas and @stevenwarejones)

**Review by SanjayVas — APPROVED** [review]
> Reviewed 3 of 4 files at r1, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/266)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @stevenwarejones)

**Review by wangyaopw — APPROVED** [review]
> Reviewed 3 of 4 files at r1, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/266)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @stevenwarejones)

**Review by stevenwarejones — APPROVED** [review]
> Reviewed 3 of 4 files at r1, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/panel-exchange-client/266)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @yunyeng)
